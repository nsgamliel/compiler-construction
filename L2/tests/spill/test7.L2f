(:go
0
mem %var2 8 += %var2
return
)

%var2

%S
