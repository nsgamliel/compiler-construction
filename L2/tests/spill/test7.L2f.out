(:go
	0 1
	%S0 <- mem rsp 0
	mem %S0 8 += %S0
	mem rsp 0 <- %S0
	return
)
